tests/tests.js

```diff
@@ -4,42 +4,145 @@ const employee = require('./employee')
+        ],
+      }
+      const price = pricing.calculateLTDPrice(products.longTermDisability, employee, selectedOptions)
+
+      expect(price).to.equal(32.04)
+    })
+
+    it('returns the price of an LTD plan without employer contribution', () => {
+      const selectedOptions = {
+        familyMembersToCover: ['ee'],
+      }
+      const price = pricing.calculateLTDPrice(products.longTermDisability, employee, selectedOptions)
+
+      expect(price).to.equal(32.04)
+    })
+
   })
 
-  it('returns the price for a voluntary life product for an employee with a spouse', () => {
-    const selectedOptions = {
-      familyMembersToCover: ['ee', 'sp'],
-      coverageLevel: [
-        { role: 'ee', coverage: 200000 },
-        { role: 'sp', coverage: 75000 },
-      ],
-    }
-    const price = pricing.calculateProductPrice(products.voluntaryLife, employee, selectedOptions)
-
-    expect(price).to.equal(71.09)
+  describe('calculateVolLifePrice', () => {
+    it('returns the price for a plan for single employee', () => {
+      const selectedOptions = {
+        familyMembersToCover: ['ee'],
+        coverageLevel: [
+          { role: 'ee', coverage: 125000 }
+        ],
+      }
+      const price = pricing.calculateVolLifePrice(products.voluntaryLife, selectedOptions)
+
+      expect(price).to.equal(43.75)
+    })
+
+    it('returns the price for a plan for an employee with a spouse', () => {
+      const selectedOptions = {
+        familyMembersToCover: ['ee', 'sp'],
+        coverageLevel: [
+          { role: 'ee', coverage: 200000 },
+          { role: 'sp', coverage: 75000 },
+        ],
+      }
+      const price = pricing.calculateVolLifePrice(products.voluntaryLife, selectedOptions)
+
+      expect(price).to.equal(79)
+    })
   })
 
-  it('returns the price for a disability product for an employee', () => {
-    const selectedOptions = {
-      familyMembersToCover: ['ee']
-    }
-    const price = pricing.calculateProductPrice(products.longTermDisability, employee, selectedOptions)
+  describe('calculateVolLifePricePerRole', () => {
+    it('returns the price for a single employee', () => {
+      const selectedOptions = {
+        familyMembersToCover: ['ee'],
+        coverageLevel: [
+          { role: 'ee', coverage: 125000 }
+        ],
+      }
+      const price = pricing.calculateVolLifePricePerRole(selectedOptions.coverageLevel[0].role, selectedOptions.coverageLevel, products.voluntaryLife.costs)
+
+      expect(price).to.equal(43.75)
+    })
+
+    it('returns the price for a spouse', () => {
+      const selectedOptions = {
+        familyMembersToCover: ['sp'],
+        coverageLevel: [
+          { role: 'sp', coverage: 85000 }
+        ],
+      }
+      const price = pricing.calculateVolLifePricePerRole(selectedOptions.coverageLevel[0].role, selectedOptions.coverageLevel, products.voluntaryLife.costs)
 
-    expect(price).to.equal(22.04)
+      expect(price).to.equal(10.2)
+    })
```

Comment:
This second test is technically the same as the first. The `calculateVolLifePricePerRole` function is a loop so testing twice with a single role isn't testing two things. Testing first with an employee and then with an employee and a spouse would be testing two things since you'd be ensure that when the loop happens it sums things up correctly.